test/cpp/end2end/crl_provider_test.cc

```diff
@@ -53,12 +53,29 @@ namespace grpc {
 namespace testing {
 namespace {
 
+const char* kSslTsiTestCrlSupportedCredentialsDir =
```

Comment:
Why do we need to define these new constants in this file? Aren't they unused?

Reply:
Good catch, I think these snuck in during a merge